### PR TITLE
aredn: harden cron maintenance scripts

### PR DESCRIPTION
**Pull request by ae6xe**
ensure maintenance scripts execute one at
a time and never in duplication

### CODE REVIEW

File: files/usr/local/bin/clean_zombie.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-<<'LICENSE'
+true <<'LICENSE'
```
Comment (dman776) [inline files/usr/local/bin/clean_zombie.sh:2]: what is "true" doing?

File: files/usr/local/bin/clean_zombie.sh [hunk 2]
```diff
@@ -36,36 +36,43 @@ LICENSE
...
 
+zombiepid="/tmp/clean_zombie.pid"
+
+[ -e $zombiepid ] && [ -d "/proc/$(cat $zombiepid)" ] && exit
+
+echo "$$" > $zombiepid
+
 # wait for rssi_monitor and snrlog to run
-sleep 10
+sleep 20;
```
Comment (dman776) [inline files/usr/local/bin/clean_zombie.sh:46]: my concern is that this keeps the shell loaded in memory twice as long.

File: files/usr/local/bin/clean_zombie.sh (hunk 1 above)
Comment (ae6xe) [reply files/usr/local/bin/clean_zombie.sh:2]: This resolved a shellcheck warning I ran into.  This is a shell pipe command going into nothing.

File: files/usr/local/bin/clean_zombie.sh (hunk 2 above)
Comment (ae6xe) [reply files/usr/local/bin/clean_zombie.sh:46]: The shell is likely in RAM 100% (shared code) all the time.  It's function has a very lite footprint only doing a 'ps' and checking file system.   We could just remove the sleep altogether.

**Review by dman776 — APPROVED** [review]

**Pull request merged**